tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,56 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = make_org({u'name': gen_alphanumeric()})
+        cv = make_content_view({u'organization-id': org['id']})
+        product_info = make_product({u'organization-id': org['id']})
+        repos = [self._make_file_repository_upload_contents(
+            {
+                u'content-type': 'file',
+                u'organization-id': org['id'],
+                u'product-id': product_info['id']
+            }
+        ) for _ in range(2)
+        ]
+
+        [ContentView.add_repository(
+            {
+                u'id': cv['id'],
+                u'repository-id': repo['id'],
+                u'organization-id': org["name"]
+            }
+        ) for repo in repos
```

Comment:
same as above , as adding both repos here in C.V. ?

Reply:
incorporated.